app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def index
```

Comment:
Test are failing for this controller @jsssanchez

Reply:
But we don't have any test for pages

Reply:
@jsssanchez the test were autogenerated and they are failing